BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/news/NewsListingFragment.java

```diff
@@ -22,6 +23,8 @@
 
 import java.util.List;
 
+import javax.xml.datatype.Duration;
```

Comment:
Is this needed? Can't see any usage of it. 🤔

Reply:
Well spotted, I couldn't remember how to add time length to toast and that was one of the guesses 😄